### ISSUE TITLE
Create a production-ready demo app

### ISSUE THREAD

**Issue by cvl** [nodegui/react-nodegui#189]
Atm all demo apps are developer sandbox only (i.e. "works on my pc").

Suggestion to create a production level demo app (at least for MacOS): i.e. simple app, which can be signed, notarized (with complete how-to readme & necessary scripts), and accepted by gatekeeper.

**Comment by master-atul** [comment]
Yep, this is definetly a critical issue.
I can try and get this done possibly during these Easter vacation days.  @cvl

**Comment by cvl** [comment]
That would be amazing :)

**Comment by cvl** [comment]
Also consider using a fork of https://github.com/mysteriumnetwork/mysterium-vpn-desktop as reference app for this.

**Comment by master-atul** [comment]
Hi @cvl 
I managed to get a signed, notarized and gatekeeper passing build ready.
I built a small demo app here: https://github.com/master-atul/DadJokes
I pretty much tweaked your script to make it more maintainable.

There was a horrible typo in nodegui packer code that led me bang my head for 2 days.
The issue was that 
`Contents/MacOs`  ---> is incorrect
`Contents/MacOS` ----> is correct 
This means any binary kept inside this (which is qode binary) will not be properly signed for some reason. But since Mac doesnt care about case sensitivity in folder names, the app package would run successfully anyway. 

So in order to fix your setup. Just upgrade to packer v1.4.1  and run the init and pack again.
or maybe just rename `MacOs` --> `MacOS`
Then use the pack and notarize scripts inside https://github.com/master-atul/DadJokes

I will get a guide up soon. But for now this should work. Let me know if you face any issues.
